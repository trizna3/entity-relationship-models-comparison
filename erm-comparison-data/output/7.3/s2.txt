projects-projects
steps-steps
values-values
attributes-attributes
experiments-experiments
REBIND_MN_TO_1NN1:S-steps-attributes
