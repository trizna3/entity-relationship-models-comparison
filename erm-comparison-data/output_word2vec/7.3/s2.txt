attributes-attributes
projects-projects
experiments-experiments
steps-steps
values-values
REBIND_MN_TO_1NN1:S-steps-attributes
