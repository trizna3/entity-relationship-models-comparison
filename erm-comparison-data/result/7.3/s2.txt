Mapping 5/5 pairs, 0 additional pairs.
Missing pairs: -
Additional pairs: -

Mapping  1/1 transformations, 0 additional transformations.
Missing transformations: -
Additional transformations: -
