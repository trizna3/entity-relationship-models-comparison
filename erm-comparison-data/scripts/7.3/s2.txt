entity set projects {
	"name"
	"date"
};

entity set experiments {
	"name"
	"date"
};

entity set steps {
	"name"
	"date"
};

entity set attributes {
	"name"
};

association between 
{
	projects ONE
	experiments MANY
}{};

association between 
{
	experiments ONE
	steps MANY
}{};

association "values" between 
{
	steps MANY
	attributes MANY
}{
	"value"
};



