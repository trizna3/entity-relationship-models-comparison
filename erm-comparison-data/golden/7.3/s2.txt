projects-projects
experiments-experiments
steps-steps
values-values
attributes-attributes
REBIND_MN_TO_1NN1:S-steps-attributes